test/scss/test.scss

```diff
@@ -6,6 +6,10 @@
 // Error Override
 $_true-error-output-override: true;
 
+// sass behaves slightly differently than node-sass
+// This flag will help adjust tests when needed
+$_dart-sass: false !default;
```

Comment:
I'd rather we find a way to make things work consistently on our end, and let users handle different implementations. I'd have to dig deeper to see how that might play out for these cases – and I'm open to ideas. But I'd rather not be testing different Sass implementations here.

Reply:
Fair point. I did it like that to check that everything would run fine with `sass`, which was my main use case. Understandable that it's not in the scope of `sass-true` to check every SASS implementation. I'll update the tests and use a spy on a JS function to verify that `runSass` does run the SASS implementation it is provided.

Actually, in terms of JS API, I was thinking it's not that great to merge that new option in the middle of the options passed to SASS. However, adding a 4th parameter doesn't sound that amazing either. Wondering if you'd be open to a breaking change and have `runSass` called like this:

```js
sassTrue.runSass({filename: 'path/to/some.scss'}, {describe: describe, it: it, sassEngine: require('sass')}
```

Newer versions of node would even allow this more succinct version, as far as I understand (to be confirmed, though):

```js
sassTrue.runSass({filename: 'path/to/some.scss'},{describe,it, sassEngine: require('sass')}
```